### PR TITLE
ci(circle): added codecov

### PR DESCRIPTION
**Pull request by Contrevien**

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -104,3 +131,11 @@ describe('<NetworkStateNotifies /> props check', () => {
     }, 2000);
   });
 });
+
+describe('<NetworkStateNotifier /> unmount check', () => {
+  const wrapper = shallow(<NetworkStateNotifier />);
+
+  it('should remove the checker after unmount', () => {
+    wrapper.unmount();
```
Comment (Kameshwaran) [inline src/index.test.js]: There is no expect?

Comment (Contrevien) [reply src/index.test.js]: This one was for code coverage. It said the code was never going to that function

Comment (Kameshwaran) [reply src/index.test.js]: ok. atleast you can add one expect to check the presence of the DOM after unmounting right?

Comment (Contrevien) [reply src/index.test.js]: Ok I will do that

**Pull request merged**